### DISCUSSION TITLE
A better help source file

### DISCUSSION THREAD

**Discussion by rousseldenis** [GrandOrgue/grandorgue#777] [Developers]
The current help xml file is quite huge and difficult to read/modify.

We maybe should enhance it.

**Comment by jason-barba** [comment]
I have (for my own learning) written an introduction to the structure of the ODF:
https://sourceforge.net/p/ourorgan/discussion/962124/thread/ef4b0f89c5/dfd4/9a8b/attachment/GO_OFD_architecture.pdf.
